CONTRIBUTING.adoc

```diff
@@ -20,6 +20,53 @@ Thank you for your contributions! Please follow this process to submit a patch:
 NOTE: If you submit a new add-on, the add-on must be added as a sub-directory in the *_/add-ons/_* sub-directory of the repository.
 See the documentation for information about link:https://docs.openshift.org/latest/minishift/using/addons.html#add-ons-overview[file structure and content format].
 
+[[testing]]
+== Testing Add-ons
+
+This repository uses integration test suite from `minishift/minishift` repository to test the functionality of the add-ons.
+The test suite is written in Go and uses Gherkin language for definition of test sets.
+This attitude allows add-on contributors to write tests effectively using predefined understandable sentences, for example:
+
+```gherkin
+  Scenario: Applying the CORS add-on
+    Given Minishift has state "Running"
+     When executing "minishift addons apply cors" succeeds
+     Then stdout should contain "CORS is now allowed from any address"
+```
```

Comment:
`Note` is needed to following the alignment pattern of `Given`, `When` and `Then` for readability IMO. Otherwise, this would be our first review comment :wink: